### PR TITLE
minor fixes and optimizations

### PR DESCRIPTION
**Pull request by brainfoolong**
Hi, just give you a little push in code readability and optimization. Also fixed a php error because of undefined variable.

### CODE REVIEW

**Comment by Cerdic** [comment]
Checked this PR today, it's safe and ready to be pulled

**Comment by mahagr** [comment]
Do you have a branch with all your changes applied?

**Comment by Cerdic** [comment]
Yes i'm putting everything all together here https://github.com/Cerdic/scssphp/tree/maint/leafo/master but this is a work in progress, I'm still working on the #627 integration wich is at the moment coming with bugs

**Comment by robocoder** [comment]
I cherry-picked the commits from this PR with the exception of 4cc12d6f. Currently, we use explicit `use` statements. In practice, I find this makes some refactoring easier (e.g., when changing the namespace of a class).

File: src/Compiler.php [hunk 1]
```diff
@@ -2004,7 +1999,7 @@ protected function shouldEval($value)
         switch ($value[0]) {
             case Type::T_EXPRESSION:
                 if ($value[1] === '/') {
-                    return $this->shouldEval($value[2], $value[3]);
```
Comment (robocoder) [inline src/Compiler.php]: This isn't just a code smell. It looks like a bug.

**Pull request closed without merging**